spring-data-jdbc/src/main/java/org/springframework/data/jdbc/repository/query/Query.java

```diff
@@ -43,7 +44,14 @@
 	String value();
 
 	/**
-	 * Optional {@link RowMapper} to use to convert the result of the query to domain class instances.
+	 * Optional {@link RowMapper} to use to convert the result of the query to domain class instances. Cannot be used
+	 * along with {@link #resultSetExtractorClass()} only one of the two can be set.
```

Comment:
New members should have `@since 1.1` tag. Is `Cannot be used along with…` still valid?